defaults/config.js

```diff
@@ -150,6 +150,13 @@ module.exports = {
 	// This value is set to `2048` kilobytes by default.
 	prefetchMaxImageSize: 2048,
 
+	// ### `prefetchMaxSearchSize`
+	//
+	// This value controls the maximum request size made to find the link preview
+	// opengraph tags. For some sites like YouTube this can easily exceed 300KB
+	// set to 50 kilobytes as to not generate a lot of traffic on the server
```

Comment:
Can this be something like:
// ### `prefetchMaxSearchSize`
//
// This value controls the maximum request size made to find the link preview
// Open Graph tags. For some sites like YouTube this can easily exceed 300 kilobytes.
//
// This value is set to `50` kilobytes by default.

Reply:
ack, done